### ISSUE TITLE
Show version in desktop files generated by appiamged

### ISSUE THREAD

**Issue by probonopd** [AppImage/AppImageKit#746]
#672 has been merged, so that version handling is now available in libappimage. Now we should make use of it in `appimaged`.

https://github.com/TheAssassin/AppImageLauncher/issues/8#issuecomment-380622507

**Comment by probonopd** [comment]
Yes, what I am looking for is that appimaged puts a menu entry named "Firefox 52.0" "Firefox [52.0]" (I prefer without brackets)

**Comment by TheAssassin** [comment]
The current implementation uses normal brackets (`(<version>)`).
